blender/arm/logicnode/draw/LN_draw_video.py

```diff
@@ -0,0 +1,52 @@
+from arm.logicnode.arm_nodes import *
+
+
+class DrawVideoNode(ArmLogicTreeNode):
+    """Draws a Video.
+
+    @input Draw: Activate to draw the Video on this frame. The input must
+        be (indirectly) called from an `On Render2D` node.
+    @input Video: The filename of the Video.
+    @input Color: The color that the Video's pixels are multiplied with.
+    @input Left/Center/Right: Horizontal anchor point of the Video.
+        0 = Left, 1 = Center, 2 = Right
+    @input Top/Middle/Bottom: Vertical anchor point of the Video.
+        0 = Top, 1 = Middle, 2 = Bottom
+    @input X/Y: Position of the anchor point in pixels.
+    @input Width/Height: Size of the Video in pixels.
+    @input Angle: Rotation angle in radians. Video will be rotated cloclwiswe
+        at the anchor point.
+
+    @output Out: Activated after the Video has been drawn.
```

Comment:
This sentence is a bit ambiguous, it means "after each call of drawVideo()" but it can be understood as "after the entire video has finished". Better word this differently, e.g. "Activated after each drawn frame from the video".

Generally, it might make sense to explain that "Draws a video" (first sentence in the docstring) means "draw the next frame from the video" (if that is correct, the JS documentation on the related functions is a bit sparse in that regard).